addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -23,6 +26,13 @@
  * @author Mark Hilbush - Initial contribution
  */
 public class BigAssFanHandlerFactory extends BaseThingHandlerFactory {
+    private Logger logger = LoggerFactory.getLogger(BigAssFanHandlerFactory.class);
+
+    @Override
+    protected void activate(ComponentContext componentContext) {
```

Comment:
It's the only way I've found to get the binding version number so that I can log it at startup.  Would be nice if there was a framework call to get the binding version.  Of course, maybe there is already and I just haven't found it.  ;-)